doc/field_guide__rr_lyrae.md

```diff
@@ -0,0 +1,47 @@
+## RR Lyrae (rrlyr)
+RR Lyrae are pulsating horizontal branch dA/dF stars with a mass 0.5 of Solar masses.
+They are part of the main 'instability strip' in the HR-diagram,
+where stars pulsate due to the double ionisation of He.
+The two main subtypes are `ab` and `c`,

```

Comment:
Should other known sub-classes be mentioned (I know this is just an example and our first, but since this is to be a template ...)

Reply:
Up to the experts, I don't have a preference. Perhaps once this is merged file an issue and assign it to @janvanroestel?